bundles/org.openhab.ui/web/src/pages/settings/rules/script/blockly-editor.vue

```diff
@@ -368,6 +367,7 @@
               <shadow type="oh_item" />
             </value>
           </block>
+          <block type="oh_item" />
```

Comment:
I would keep this green block where it was because it sort of serves as "header" of the group of block types related to items, the same way the green "thing picker" is the header of the block (or put these at the end of each group).

Reply:
I hear what you saying and I was thinking about it but I noticed that many people use these and think it is the state but it actually is "not more than a string". I would then go for putting both to the end which is probably a good compromise.

Reply:
Thinking about this further I would also put the "send command/update state" block immediately after "get state" as they're somewhat related and probably the most used.